server/keycloak.js

```diff
@@ -22,13 +22,20 @@ const defaults = {
 class Keycloak {
   // Wrapper class around keycloak-connect
   constructor() {
+    const isLocal = process.env.KEYCLOAK_AUTH_URL.includes('local');
     this.realm = process.env.KEYCLOAK_REALM;
     this.authUrl = process.env.KEYCLOAK_AUTH_URL;
     this.clientNameFrontend = process.env.KEYCLOAK_FE_CLIENTID;
     this.clientNameBackend = process.env.KEYCLOAK_API_CLIENTID;
-    this.clientSecretBackend = process.env.KEYCLOAK_API_SECRET;
-    this.serviceAccountUsername = process.env.KEYCLOAK_SA_USERNAME;
-    this.serviceAccountPassword = process.env.KEYCLOAK_SA_PASSWORD;
+    this.clientSecretBackend = isLocal
```

Comment:
what is the exact requirement to have separate set of env values here?

Reply:
a lot of the necessary environment variables differ between the local and remote instances. This pattern seemed like the easiest way to minimize the logic of adapting between those cases.